inc/compatibilities/master_slider.php

```diff
@@ -27,10 +27,21 @@ public function register() {
 		add_filter(
 			'optml_lazyload_bg_selectors',
 			function ( $all_watchers ) {
-				$all_watchers[] = '.master-slider';
+				$all_watchers = array_merge( $all_watchers, ['.master-slider'] );
 				return $all_watchers;
 			}
 		);
+		add_filter(
+			'optml_dont_replace_url',
+			function ( $arg, $url = null ) {
```

Comment:
` function( $old, $url ) {`
schimba semnatura functiei sa fie aceiasi ca in documentatie

Reply:
ok